src/components/all-on-one-form-vc-button/all-on-one-form-vc-button.ts

```diff
@@ -0,0 +1,130 @@
+import { VehicleCheckProvider, vCheckType } from './../../providers/vehicle-check/vehicle-check';
+import { Component, ElementRef } from '@angular/core';
+import { HazardRecorderProvider } from '../../providers/hazard-recorder/hazard-recorder';
+import { CustomHammerConfigProvider } from '../../providers/custom-hammer-config/custom-hammer-config';
+
+export interface IVehicleCheck {
+  complete?: boolean;
+  faultType?: string;
+}
+
+declare const Hammer: any;
+@Component({
+  selector: 'all-on-one-form-vc-button',
+  templateUrl: 'all-on-one-form-vc-button.html'
+})
+export class AllOnOneFormVcButtonComponent {
+  tellMe: IVehicleCheck = {};
+  showMe: IVehicleCheck = {};
+  mc: any;
```

Comment:
mc?

Reply:
refers to the Hammer manager for press and hold functionality that has been copied from other components. I'm sure what it should stand for but it's required